src/test/java/com/mysql/cj/jdbc/ha/ca/plugins/ConnectionPluginManagerTest.java

```diff
@@ -0,0 +1,82 @@
+package com.mysql.cj.jdbc.ha.ca.plugins;
+
+import com.mysql.cj.conf.PropertySet;
+import com.mysql.cj.jdbc.ha.ca.ClusterAwareConnectionProxy;
+import com.mysql.cj.log.Log;
+import com.mysql.cj.log.NullLogger;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import software.aws.rds.jdbc.mysql.Driver;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.spy;
+
+@TestMethodOrder(MethodOrderer.Alphanumeric.class)
```

Comment:
Why does these tests need to run in alphanumeric order?

Reply:
Test removed.